barretenberg/cpp/src/barretenberg/bb/main.cpp

```diff
@@ -218,14 +253,17 @@ void contract(const std::string& output_path, const std::string& vk_path)
  *
  *
  * @param proof_path Path to the file containing the serialized proof
- * @param vk_path Path to the file containing the serialized verification key
  * @param output_path Path to write the proof to
  */
-void proofAsFields(const std::string& proof_path, std::string const& vk_path, const std::string& output_path)
+void proofAsFields(const std::string& proof_path, const std::string& output_path)
```

Comment:
Even without splitting the public inputs, we could have removed the need for vk by getting the proof size and minusing the size of the proof then dividing by 32 to get the number of public inputs